src/textual/layout.py

```diff
@@ -1,23 +1,37 @@
 from .widget import Widget
 
 
-class Vertical(Widget):
-    """A container widget to align children vertically."""
+class Container(Widget):
```

Comment:
A base class for common styles, and for containers where you might want to dynamically tweak the layout.